Gemfile

```diff
@@ -111,6 +111,8 @@ gem 'demoji', "~> 0.0.5"
 gem 'faraday', '~> 0.9.1'
 gem 'faraday_middleware', '~> 0.10.0'
 
+gem "react_on_rails", "~> 5"
```

Comment:
Maybe a stricter versioning here? `bundle update` may now update react_on_rails to `6`, which may not be desired.